README.md

```diff
@@ -174,63 +174,94 @@ If all goes well, you will soon seen the end-to-end tests running via your Chrom
+
+```
+cd ~/cbioportal-docker-compose
+docker-compose run --rm cbioportal sh -c '
+    cd /cbioportal/core/src/main/scripts/ \
+    && ./importGenePanel.pl --data /cbioportal/core/src/test/scripts/test_data/study_es_0/data_gene_panel_testpanel1.txt \
+    && ./importGenePanel.pl --data /cbioportal/core/src/test/scripts/test_data/study_es_0/data_gene_panel_testpanel2.txt \
+    && ./importGenesetData.pl --data /cbioportal/core/src/test/resources/genesets/study_es_0_genesets.gmt --new-version msigdb_6.1 \
+    && ./importGenesetHierarchy.pl --data /cbioportal/core/src/test/resources/genesets/study_es_0_tree.yaml'
 ```
 
-3. Running tests can be executed (a) on the local system or (b) in the docker environment;
+Restart cBioPortal
+```
+docker-compose restart cbioportal
+```
+
+3. Load study_es_0 of the deployed (!) backend version and all studies in [end-to-end-test/local/studies](end-to-end-test/local/studies). At the moment of this writing:
 
-(a) local system (best for test development)
-```bash
-export CBIOPORTAL_URL='http://localhost:8081'
-$TEST_HOME/local/runtime-config/setup_local_context.sh -j -d -p -e # remove flags to exclude specific stages if desired (see below)
-cd $PORTAL_SOURCE_DIR
-./end-to-end-test/local/runtime-config/run_local_screenshot_test.sh # (repeat this step while developing)
 ```
+docker-compose run --rm  -v <e2e_study_dir>:/studies cbioportal sh -c '
+    cd /cbioportal/core/src/main/scripts/importer \
+    && ./cbioportalImporter.py -s /cbioportal/core/src/test/scripts/test_data/study_es_0 \
+    && ./cbioportalImporter.py -s /studies/genepanel_test_study'
+    # add other studies if present
+```
+
+:info: The `<e2e_study_dir>` entry refer to the absolute path to [end-to-end-test/local/studies](end-to-end-test/local/studies) on the host system.
+
+4. Add `export CBIOPORTAL_URL="http://localhost:8080"` to `/env/custom.sh`.
+
+5. In a terminal, install webdriver-manager:
 
-(b) docker environment (best for making reference screenshots)
 ```bash
-export CBIOPORTAL_URL='http://cbioportal:8080'
-yarn
-yarn build
-cd $TEST_HOME/local/docker
-./setup_docker_containers.sh -p -d -e # remove flags to exclude specific stages if desired (see below)
-cd $PORTAL_SOURCE_DIR
-$TEST_HOME/local/runtime-config/run_container_screenshot_test.sh
+yarn global add webdriver-manager
 ```
 
-4. When finished reclaim ports by killing selenium and node servers:
-```bash
-lsof -n -i4TCP:3000 && lsof -n -i4TCP:3000 | grep LISTEN | awk '{ print $2 }' | xargs kill # reclaims port 3000
-lsof -n -i4TCP:4444 && lsof -n -i4TCP:4444 | grep LISTEN | awk '{ print $2 }' | xargs kill # reclaims port 4444
+:warning: Add path to webdriver-manager installation (needed on Ubuntu Linux):
+```
+export PATH=$PATH:$(yarn global dir)/node_modules/webdriver-manager/bin
+```
+
+and start webdriver-manager:
 ```
+webdriver-manager update
+webdriver-manager start
+```
+
+6. In a second terminal, build and serve the frontend version under test:
```

Comment:
I might add here that this the normal setup for portal development, so most people should already have this set up.   i.e. if one is already developing portal, it doesn't need to be separate

Reply:
I know. But under step 7 there are slight differences so I reasoned it to be better to show all steps. You think we should remove these steps?

Reply:
Not sure what you mean here.  I would just say 
```
If you do not already have the frontend dev server running at localhost:3000: In a second terminal ...
```

Reply:
@alisman I updated the test to:

```
6. If not already running at _localhost:3000_, open a second terminal and start the frontend dev server:
```